modules/openapi-generator/src/main/resources/go-server/controller-api.mustache

```diff
@@ -329,7 +339,25 @@ func (c *{{classname}}Controller) {{nickname}}(w http.ResponseWriter, r *http.Re
 	{{^items.isDouble}}
 	{{^items.isLong}}
 	{{^items.isInteger}}
-	{{paramName}}Param := strings.Split(query.Get("{{baseName}}"), ",")
+	{{^items.isEnumRef}}
+	var {{paramName}}Param [] string
+	if query.Has("{{baseName}}") {
+		{{paramName}}Param = strings.Split(query.Get("{{baseName}}"), ",")
+	}
+	{{/items.isEnumRef}}
+	{{#items.isEnumRef}}
+	var {{paramName}}Param []{{items.dataType}}
+	if query.Has("{{baseName}}") {
+		for _, param := range strings.Split(query.Get("{{baseName}}"), ",") {
```

Comment:
Could we do `strings.Split(query.Get("{{baseName}}"), ",")` first then create var `{{paramName}}Param` with the exact size. make([]{{items.dataType}}, size)